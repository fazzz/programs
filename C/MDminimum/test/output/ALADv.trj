--2.335  --3.752  -0.146  --2.396  --2.652  -0.123  --2.903  --2.324  -1.019  
--2.886  --2.306  --0.746  --0.965  --2.106  -0.123  --0.001  --2.862  -0.128  --0.840  
--0.772  -0.128  --1.658  --0.178  -0.140  -0.465  --0.130  -0.137  -1.027  --0.437  
-1.025  -1.270  --0.522  --1.103  -0.753  --0.217  --2.016  -2.226  --0.040  --1.076  
-1.420  --1.592  --1.131  -0.319  -1.388  -0.129  --0.793  -1.911  -0.131  -1.450  
-2.088  -0.130  -2.355  -1.605  -0.139  -1.454  -3.543  -0.126  -0.435  -3.937  
-0.138  -2.002  -3.862  -0.978  -1.968  -3.934  --0.765  --2.340  --3.804  --0.092  --2.323  --2.684  --0.178  --3.057  --2.288  -0.551  
--2.633  --2.325  --1.145  --0.953  --2.127  -0.098  --0.062  --2.864  -0.436  --0.781  
--0.832  --0.157  --1.568  --0.250  --0.430  -0.525  --0.140  --0.046  -1.204  --0.562  
-0.698  -1.321  --0.338  --1.325  -0.794  -0.056  --2.224  -2.270  -0.156  --1.158  
-1.523  --1.408  --1.428  -0.307  -1.332  -0.248  --0.837  -1.757  -0.429  -1.413  
-2.146  -0.315  -2.387  -1.760  -0.335  -1.357  -3.624  -0.225  -2.194  -4.057  
-0.781  -1.505  -3.815  --0.808  -0.410  -4.083  -0.647  --2.318  --3.680  -0.540  --2.335  --2.740  --0.087  --3.199  --2.059  -0.108  
--2.495  --3.001  --1.170  --0.921  --2.083  -0.093  --0.133  --2.809  -0.643  --0.620  
--0.855  --0.404  --1.341  --0.397  --0.948  -0.634  --0.136  --0.312  -1.255  --0.558  
-0.488  -1.475  --0.173  --1.569  -0.979  -0.214  --2.470  -2.438  -0.297  --1.461  
-1.672  --1.226  --1.769  -0.243  -1.315  -0.130  --0.951  -1.649  -0.230  -1.273  
-2.216  -0.288  -2.216  -1.828  -0.050  -1.337  -3.509  -0.989  -1.212  -4.320  
-0.263  -0.327  -3.727  -1.545  -2.239  -3.705  -1.558  --2.233  --3.693  -0.509  --2.219  --2.800  --0.090  --3.006  --2.115  -0.184  
--2.461  --3.094  --1.092  --0.821  --2.187  -0.048  -0.124  --2.800  -0.463  --0.696  
--0.913  --0.330  --1.533  --0.432  --0.646  -0.562  --0.096  --0.324  -1.292  --0.459  
-0.435  -1.174  --0.006  --1.740  -0.483  -0.504  --2.423  -2.085  -0.553  --1.710  
-1.352  --1.016  --2.107  -0.243  -1.346  -0.128  --0.806  -1.825  --0.296  -1.190  
-2.033  -0.836  -2.026  -1.453  -1.044  -1.239  -3.475  -1.286  -0.271  -3.946  
-1.273  -1.569  -3.513  -2.368  -2.006  -3.945  -0.742  --1.887  --3.797  --0.328  --2.033  --2.809  --0.082  --2.360  --2.704  -0.921  
--2.803  --2.350  --0.758  --0.698  --2.078  --0.080  -0.288  --2.705  -0.115  --0.728  
--0.804  --0.223  --1.632  --0.329  --0.233  -0.510  --0.046  --0.234  -1.172  --0.487  
-0.571  -0.997  --0.249  --1.731  -0.284  -0.265  --2.357  -1.999  -0.290  --1.911  
-1.060  --1.318  --2.050  -0.297  -1.402  -0.067  --0.611  -2.018  --0.491  -1.042  
-1.913  -1.026  -1.784  -1.360  -1.453  -0.944  -3.255  -1.588  -0.668  -3.249  
-2.617  -1.930  -3.696  -1.555  -0.353  -3.879  -0.947  --2.153  --3.668  --0.306  --2.062  --2.646  --0.018  --2.342  --2.653  -1.037  
--2.851  --2.107  --0.613  --0.624  --2.094  --0.177  -0.346  --2.798  -0.041  --0.636  
--0.808  --0.460  --1.500  --0.301  --0.411  -0.572  --0.075  --0.344  -1.287  --0.513  
-0.391  -1.210  -0.003  --1.796  -0.508  -0.425  --2.510  -2.127  -0.642  --1.744  
-1.551  --0.962  --2.185  -0.232  -1.329  -0.151  --0.697  -2.018  --0.275  -0.943  
-1.803  -1.148  -1.701  -1.224  -1.479  -0.750  -3.143  -1.656  -1.696  -3.695  
-1.719  -0.096  -3.720  -0.974  -0.254  -3.110  -2.630  --1.995  --3.800  -0.146  --2.009  --2.869  --0.473  --2.913  --2.303  --0.191  
--2.092  --3.060  --1.552  --0.659  --2.173  --0.246  -0.306  --2.849  -0.086  --0.623  
--0.868  --0.442  --1.512  --0.484  --0.703  -0.514  --0.003  --0.363  -1.242  --0.474  
-0.380  -1.290  -0.173  --1.706  -0.701  -0.492  --2.497  -2.035  -0.945  --1.561  
-1.783  --0.694  --2.031  -0.187  -1.382  -0.218  --0.903  -2.004  -0.108  -1.132  
-1.978  -0.936  -1.983  -1.433  -1.114  -0.775  -3.128  -1.804  -0.279  -3.935  
-1.262  --0.012  -2.808  -2.453  -1.717  -3.503  -2.368  --1.942  --3.892  --0.327  --1.835  --3.013  --0.940  --2.746  --2.477  --1.056  
--1.613  --3.384  --1.950  --0.663  --2.203  --0.434  -0.259  --2.905  --0.024  --0.646  
--0.891  --0.507  --1.392  --0.419  --0.967  -0.488  -0.011  --0.191  -1.237  --0.522  
-0.451  -1.280  -0.418  --1.441  -0.696  -1.129  --2.034  -2.208  -0.977  --1.117  
-1.520  --0.508  --1.969  -0.080  -1.289  -0.508  --1.045  -1.802  -0.452  -1.048  
-1.975  -1.084  -1.930  -1.492  -1.230  -0.992  -3.398  -1.506  -0.248  -3.578  
-2.239  -1.884  -3.726  -1.981  -0.778  -3.942  -0.535  --1.722  --3.812  --1.493  --1.880  --2.768  --1.155  --2.630  --2.672  --0.376  
--2.227  --2.193  --1.995  --0.617  --2.169  --0.560  -0.395  --2.861  --0.335  --0.647  
--0.860  --0.419  --1.539  --0.461  --0.635  -0.562  --0.103  -0.033  -1.131  --0.629  
-0.837  -1.387  -0.102  --1.244  -0.770  -0.655  --1.998  -2.344  -0.604  --1.050  
-1.661  --0.898  --1.620  -0.122  -1.301  -0.519  --1.112  -1.520  -0.535  -1.039  
-2.259  -0.890  -2.013  -1.940  -0.835  -0.768  -3.474  -1.729  -1.673  -4.065  
-1.877  --0.008  -4.105  -1.302  -0.396  -3.096  -2.674  --1.982  --3.535  --1.755  --1.889  --2.776  --0.919  --1.996  --3.295  -0.059  
--2.693  --2.114  --1.019  --0.584  --2.055  --0.775  -0.436  --2.725  --0.662  --0.565  
--0.789  --0.603  --1.445  --0.343  --0.554  -0.578  --0.079  -0.023  -1.047  --0.691  
-0.832  -1.583  -0.236  --1.067  -1.051  -0.769  --1.871  -2.409  -0.811  --0.818  
-1.906  --0.683  --1.530  -0.039  -1.196  -0.614  --1.098  -1.485  -0.375  -0.869  
-2.010  -1.289  -1.652  -1.532  -1.676  -0.668  -3.389  -1.683  --0.436  -3.585  
-1.846  -1.193  -3.721  -2.584  -0.873  -4.005  -0.833  