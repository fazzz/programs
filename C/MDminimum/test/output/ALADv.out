E_t    = 8.078178e+00 
KE     = 2.082706e+01 
KEv    = 8.035431e-02 
PEv    = 9.833785e-01 
p_tot  = -1.381262e+01 
T      = 3.175894e+02 
E_t    = 8.220703e+00 
KE     = 2.338810e+01 
KEv    = 4.630697e-02 
PEv    = -4.739522e+00 
p_tot  = -1.047418e+01 
T      = 3.566424e+02 
E_t    = 8.423923e+00 
KE     = 2.427016e+01 
KEv    = 2.385340e-01 
PEv    = -1.820371e+01 
p_tot  = 2.118939e+00 
T      = 3.700927e+02 
E_t    = 8.174998e+00 
KE     = 2.582483e+01 
KEv    = 3.023357e-01 
PEv    = -8.811778e+00 
p_tot  = -9.140387e+00 
T      = 3.937997e+02 
E_t    = 8.418035e+00 
KE     = 2.132479e+01 
KEv    = 3.596571e-02 
PEv    = -1.039642e+01 
p_tot  = -2.546309e+00 
T      = 3.251792e+02 
E_t    = 8.145676e+00 
KE     = 2.650293e+01 
KEv    = 1.078104e+00 
PEv    = -1.256825e+01 
p_tot  = -6.867109e+00 
T      = 4.041399e+02 
E_t    = 8.420323e+00 
KE     = 2.492340e+01 
KEv    = 1.155370e-01 
PEv    = -1.255191e+01 
p_tot  = -4.066707e+00 
T      = 3.800540e+02 
E_t    = 8.290331e+00 
KE     = 2.910163e+01 
KEv    = 1.573727e-01 
PEv    = -1.475133e+01 
p_tot  = -6.217342e+00 
T      = 4.437672e+02 
E_t    = 8.156688e+00 
KE     = 2.652517e+01 
KEv    = 9.030678e-01 
PEv    = -1.076693e+01 
p_tot  = -8.504618e+00 
T      = 4.044791e+02 
E_t    = 8.360948e+00 
KE     = 2.043864e+01 
KEv    = 4.359044e-01 
PEv    = -1.052398e+01 
p_tot  = -1.989621e+00 
T      = 3.116664e+02 
